Examples/Swift/ViewController.swift

```diff
@@ -159,6 +159,7 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         
         viewController.routeController.snapsUserLocationAnnotationToRoute = true
         viewController.voiceController?.volume = 0.5
+        viewController.navigationDelegate = self
```

Comment:
@1ec5 is this coming in too late? I'm not hitting the delegate methods down below.